library_management/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import Book, BookIssue, BookRequest, NewBookTicket
+
+
+class BookSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Book
+        fields = '__all__'
```

Comment:
Returning all fields is not a good practice because down the line if you need to add sensitive fields every related serializer would serialize that as well. You should list every  field instead.

Reply:
Got it.